assignments/objects.js

```diff
@@ -15,30 +15,66 @@ let example = {
   "name": "Example",
   "email": "examples@you.edu",
   "gender": "F"
-}
+};
 
 // Write your intern objects here:
-
+let mitzi = {
+	"id": 1,
+	"name": "Mitzi",
+	"email": "mmelloy0@psu.edu",
+	"gender": "F"
+};
+
+let kennan = {
+	"id": 2,
+    "name": "Kennan",
+    "email": "kdiben1@tinypic.com",
+    "gender": "M"
+};
+
+let keven = {
+	"id": 3,
+    "name": "Keven",
+    "email": "kmummery2@wikimedia.org",
+    "gender": "M"
+};
+
+let gannie = {
+	"id": 4,
+	"name": "Gannie",
+	"email": "gmartinson3@illinois.edu",
+	"gender": "M"
+};
+
+let antonietta = {
+	"id": 5,
+	"name": "Antonietta",
+	"email": "adaine5@samsung.com",
+	"gender": "F"
+};
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
-
+console.log(mitzi.name);
 // Kennan's ID
-
+console.log(kennan.id);
 // Keven's email
-
+console.log(keven.email);
 // Gannie's name
-
+console.log(gannie.name);
 // Antonietta's Gender
+console.log(antonietta.gender);
 
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
+console.log(kennan.speak());
```

Comment:
So, here try creating a function that return the string "Hello, my name is Kennan!"